lib/defaults.js

```diff
@@ -2,260 +2,111 @@
-    //'no-eval': 2,                    // http://eslint.org/docs/rules/no-eval
-    //'no-extend-native': 2,           // http://eslint.org/docs/rules/no-extend-native
-    //'no-extra-bind': 2,              // http://eslint.org/docs/rules/no-extra-bind
-    //'no-fallthrough': 2,             // http://eslint.org/docs/rules/no-fallthrough
-    //'no-floating-decimal': 2,        // http://eslint.org/docs/rules/no-floating-decimal
-    //'no-implied-eval': 2,            // http://eslint.org/docs/rules/no-implied-eval
-    //'no-lone-blocks': 2,             // http://eslint.org/docs/rules/no-lone-blocks
-    //'no-loop-func': 2,               // http://eslint.org/docs/rules/no-loop-func
-    //'no-multi-str': 2,               // http://eslint.org/docs/rules/no-multi-str
-    //'no-native-reassign': 2,         // http://eslint.org/docs/rules/no-native-reassign
-    //'no-new': 2,                     // http://eslint.org/docs/rules/no-new
-    //'no-new-func': 2,                // http://eslint.org/docs/rules/no-new-func
-    //'no-new-wrappers': 2,            // http://eslint.org/docs/rules/no-new-wrappers
-    //'no-octal': 2,                   // http://eslint.org/docs/rules/no-octal
-    //'no-octal-escape': 2,            // http://eslint.org/docs/rules/no-octal-escape
-    'no-param-reassign': 0,          // http://eslint.org/docs/rules/no-param-reassign
-    //'no-proto': 2,                   // http://eslint.org/docs/rules/no-proto
-    //'no-redeclare': 2,               // http://eslint.org/docs/rules/no-redeclare
-    //'no-return-assign': 2,           // http://eslint.org/docs/rules/no-return-assign
-    //'no-script-url': 2,              // http://eslint.org/docs/rules/no-script-url
-    //'no-self-compare': 2,            // http://eslint.or//g/docs/rules/no-self-compare
-    //'no-sequences': 2,               // http://eslint.org/docs/rules/no-sequences
-    //'no-throw-literal': 2,           // http://eslint.org/docs/rules/no-throw-literal
-    //'no-with': 2,                    // http://eslint.org/docs/rules/no-with
-    //'radix': 2,                      // http://eslint.org/docs/rules/radix
-    'vars-on-top': 0,                // http://eslint.org/docs/rules/vars-on-top
-    'wrap-iife': [2, 'any'],         // http://eslint.org/docs/rules/wrap-iife
-    //'yoda': 2,                       // http://eslint.org/docs/rules/yoda
+    'consistent-return': [1, { treatUndefinedAsUnspecified: true}],          
+    'eqeqeq': 2,                     
+    'no-eq-null': 2,                 
+    'no-param-reassign': 0,          
+    'vars-on-top': 0,                
+    'wrap-iife': [2, 'any'],         
 
     /**
      * Style
      */
-    /*'brace-style': [                 // http://eslint.org/docs/rules/brace-style
-      2,
-      '1tbs',
-      { 'allowSingleLine': true }
-    ],*/
-    'camelcase': [2, {               // http://eslint.org/docs/rules/camelcase
+    'camelcase': [2, {               
```

Comment:
@ctang18, [ESLint provides a way](http://eslint.org/docs/user-guide/configuring#disabling-rules-with-inline-comments) to disable rule errors/warnings for a particular section of code using comments, which we can use in this scenario.

Note that as a team we've agreed to specify which rule(s) we are disabling to aid in readability and prevent unintended lint violations from not getting flagged. See #24.